package.json

```
@@ -34,7 +34,7 @@
     "docs:typescript:formatted": "cross-env BABEL_ENV=development babel-node --extensions \".tsx,.ts,.js\" ./docs/scripts/formattedTSDemos",
     "docs:mdicons:synonyms": "cross-env BABEL_ENV=development babel-node --extensions \".tsx,.ts,.js,.mjs\" ./docs/scripts/updateIconSynonyms && pnpm prettier",
     "extract-error-codes": "cross-env MUI_EXTRACT_ERROR_CODES=true lerna run --concurrency 8 build:modern",
-    "rsc:build": "ts-node ./packages/rsc-builder/buildRsc.ts",
+    "rsc:build": "tsx ./packages/rsc-builder/buildRsc.ts",
```

If we don't want to depend on either, this would work as well:

```suggestion
    "rsc:build": "cross-env BABEL_ENV=development babel-node --extensions \".tsx,.ts,.js\" ./packages/rsc-builder/buildRsc.ts",
```

---

babel-node is problematic when running ESM scripts. It doesn't read tsconfig, so we're can't set module and resolution strategy (at least not as simply as in tsconfig). I'd rather use tsconfigs for this and slowly phase Babel away (to be able to use SWC in Next JS).